lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -104,10 +106,12 @@ class ExpansionContactListTile extends StatelessWidget {
   }
 
   Widget _buildDisplayName(BuildContext context, ProfileInformation? profile) {
-    if (profile != null) {
-      return _displayName(context, profile.displayname!);
+    if (isSearch && contact.displayName != null) {
+      return _displayName(context, contact.displayName!);
     } else if (contact.displayName != null) {
       return _displayName(context, contact.displayName!);
+    } else if (profile != null) {
+      return _displayName(context, profile.displayname!);
     } else {
       return const SizedBox.shrink();
     }
```

Comment:
we should trying to display well the display name first and other case should be less priority